guides/doc-Upgrading_and_Updating/topics/migrating_pulp_content.adoc

```diff
@@ -0,0 +1,71 @@
+[[migrating_pulp_content]]
```

Comment:
This looks like the Pulp 2 -> 3 migration document. It's a good practice to keep pull requests limited to a single topic. Experience tells us it reduces merge time. https://theforeman.org/2019/03/merge-time-review-complexity.html

Reply:
Not sure how that got in there.

Reply:
I'll remove that part.

Reply:
This file got into the PR by accident. It's now been removed.

Reply:
This file is still here.

Reply:
@ekohl Hopefully, cross fingers, touch-wood, this is now removed.